docs/pages/x/api/date-pickers/date-time-picker.json

```diff
@@ -3,10 +3,7 @@
     "ampm": { "type": { "name": "bool" }, "default": "utils.is12HourCycleInCurrentLocale()" },
     "ampmInClock": { "type": { "name": "bool" }, "default": "true on desktop, false on mobile" },
     "autoFocus": { "type": { "name": "bool" } },
-    "closeOnSelect": {
-      "type": { "name": "bool" },
-      "default": "`true` for desktop, `false` for mobile (based on the chosen wrapper and `desktopModeMediaQuery` prop)."
-    },
+    "closeOnSelect": { "type": { "name": "bool" }, "default": "`false`" },
```

Comment:
There's an issue with the API reference on our docs:

To have a precise definition on our page, we would have to add different descriptions for this prop according to the component. For example:

- For `DatePickers` it should be: 
  "`true` for desktop, `false` for mobile (based on the chosen wrapper and `desktopModeMediaQuery` prop)."

- For `TimePickers` it should be: 
  "`false`"

We currently already have some inconsistency regarding this prop, for example:

- [`MobileDateRangePicker`](https://mui.com/x/api/date-pickers/mobile-date-range-picker/#mobile-date-range-picker-prop-closeOnSelect) presents the description:

  "`true` for desktop, `false` for mobile (based on the chosen wrapper and `desktopModeMediaQuery` prop)."

Which is confusing since the reference is on the mobile component

The problem root relies on the fact that the docs doesn't allow different descriptions for the same prop.